src/js/pyodide.js

```diff
@@ -162,12 +161,18 @@ function fixRecursionLimit() {
  * (This can be fixed once `Firefox adopts support for ES6 modules in webworkers
  * <https://bugzilla.mozilla.org/show_bug.cgi?id=1247687>`_.)
  *
- * @param {{ indexURL : string, fullStdLib? : boolean = true }} config
+ * @param {{ indexURL : string, fullStdLib? : boolean = true, input?: () => string, print?: (text: string) => void, printErr?: (text: string) => void }} config
```

Comment:
I wonder if someday it would be worthwhile to use Typescript. Currently, this doesn't get recognized by Visual Studio Code.

Reply:
Indeed. Opened https://github.com/pyodide/pyodide/issues/1729